### PR TITLE
Add actual DCA setup

### PR DESCRIPTION
**Pull request by njacazio**
@mfaggin

### CODE REVIEW

**Comment by njacazio** [comment]
@mfaggin I realized it had no effect before, what do you think?

File: Common/Core/TrackSelectionDefaults.h [hunk 1]
```diff
@@ -26,7 +26,7 @@ TrackSelection getGlobalTrackSelection();
 
 // Default track selection requiring a particular Run 3 ITS matching
 TrackSelection getGlobalTrackSelectionRun3ITSMatch(int matching,
-                                                   TrackSelection::GlobalTrackRun3DCAxyCut passFlag = TrackSelection::GlobalTrackRun3DCAxyCut::Default);
+                                                   int passFlag = TrackSelection::GlobalTrackRun3DCAxyCut::Default);
```
Comment (mfaggin) [inline Common/Core/TrackSelectionDefaults.h:29]: did you cross check that this does not give any subtle error? In principle if it compiles it should be fine, but have you tried to run it?

**Review by mfaggin — COMMENTED** [review]
Hi @njacazio , thanks for fixing it. Apparently the `dcaSetup.value` was missing when the selections were initialized. Just a minor comment, after the reply on it I will approve it.

File: Common/Core/TrackSelectionDefaults.h (hunk 1 above)
Comment (njacazio) [reply Common/Core/TrackSelectionDefaults.h:29]: Hi, yes I'm running with this since some weeks now

**Review by mfaggin — APPROVED** [review]

**Review by ddobrigk — APPROVED** [review]
thanks!

**Pull request merged**